src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/xresilience/ChaosSpec.groovy

```diff
@@ -83,6 +105,10 @@ class ChaosSpec extends HealthCheckSpecification {
                 }.empty
             }
         }
+
+        topology.activeSwitches.each {
```

Comment:
I'd replace with asynchronous switch synchronization. But leave it as is, I'll cover it in my refactoring.